### DISCUSSION TITLE
темы

### DISCUSSION THREAD

**Discussion by jorhox** [anilibria/anilibria-winmaclinux#194] [Задать вопрос]
увидел возможность смены темы из файла, где можно темы найти и скачать?

**Comment by trueromanus** [comment]
В приложении на странице "Менеджер Тем" перейдите на вкладку "Доступные для скачивания" (нажать на кнопку с четырьмя квадратами в правом верхнем углу). Там будет список тем доступных для Вашей базовой темы которые можно скачать нажав на кнопку с облачком, после чего они станут доступны для выбора в верхнем списке для смены темы.  К сожалению там выбор не очень велик.
